.travis.yml

```diff
@@ -23,7 +23,8 @@ python:
 
 env:
     global:
-        CXXFLAGS="-O3 -Wall -Wextra -Wno-unused-parameter -Wno-empty-body -Wno-format-security" 
+        CXXFLAGS="-O3 -std=c++11 -Wall -Wextra -Wno-unused-parameter -Wno-empty-body -Wno-format-security"
+        CPPFLAGS="-DNDEBUG"
```

Comment:
This flag is required to get the best performance out of the container class as it disables the in-range checks when accessing the data. I will update the build system accordingly.